lib/Conversion/TorchOnnxToTorch/Patterns.cpp

```diff
@@ -24,12 +25,25 @@ LogicalResult OnnxCustomOpConversionPattern::matchAndRewrite(
   auto foundIt = namedHandlers.find(op.getNameAttr());
   if (foundIt == namedHandlers.end())
     return failure();
+  // If the op has an individual version, use it.
+  int64_t opDomainVersion = domainVersion;
+  if (auto attr =
+          op->getAttrOfType<IntegerAttr>("torch.onnx_meta.opset_version")) {
```

Comment:
My understanding of terminology is that the model has an `opset` version (which combines multiple ops; each individually being at that or a lower version), but an op itself has an `op` version, which is called `since_version` in the documentation (https://onnx.ai/onnx/operators/onnx__Conv.html#conv-22).
Maybe call the attribute `torch.onnx_meta.version`?

Reply:
We can go for just `version`, there is no ambiguity. The version number of an op is indeed the (sequential) opset version number in which _this specific revision_ of the op was introduced, so in that sense I also used `opset_version`.

Reply:
Both is fine for me